File: Computer Programming/C++/String/Valid Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) 
    {
        unordered_map<int,int> mps;
        unordered_map<int,int> mpt;

        for(auto it : s)
        {
            mps[it] ++;
        }    
        for(auto it : t)
        {
            mpt[it] ++;
        }

        if(mps == mpt)
        {
            return true;
        }
        return false;
    }
};

File: Computer Programming/Python/Basics/Valid Anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        
        mp1 = {}
        mp2 = {}

        s_size = len(s)
        t_size = len(t)
        for i in range(s_size):
            mp1[s[i]] = mp1.get(s[i], 0) + 1
        
        for j in range(t_size):
            mp2[t[j]] = mp2.get(t[j],0) + 1

        if mp1 == mp2:
            return True
        
        return False
